### ISSUE TITLE
A11y

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3401]
**GitHubTags:**

#A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG1.3.1;
#Screenreader; #A11yMAS;#Narrator.

**OS Details:**

Web: win11

**Environment Details:**

Application Name: Fluent Ui

**Repro Steps:**

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Components" button and hit enter.
4.Tab till "Toolbars" link on the left side bar and hit enter. 
5."Toolbars" page will open.
6. Verify Narrator is not announcing the names"bullet,underline,italic" it is announcing as button

**Actual Result**

The narrator is not announcing the names"bullet,underline,italic" it is announcing as button

**Expected Result:**

The narrator should announce the names also like bullet button, italic button, underline button.

**User Impact:**

Screen reader users will face difficulties when narrator is not announcing the names"bullet,underline,italic" it is announcing as button only.

![3401 A11y_Fluent UI_Components_Toolbar_ScreenReader](https://user-images.githubusercontent.com/89508906/198562452-4ecee4cd-bc78-4af9-b1f6-fc5a1834b0fa.gif)

**Comment by Latha2564** [comment]
Logged a bug under wrong ADO .Hence Closing the bug
